### PR TITLE
Create 2016-04-15-tbd.md

### PR DESCRIPTION
**Pull request by skeskali**
Draft of TGIF 79

### CODE REVIEW

**Comment by varjmes** [comment]
Thank you for mentioning me! :)

**Comment by gr2m** [comment]
I’m reviewing now

**Comment by gr2m** [comment]
LGTM :ok_hand:

**Comment by gr2m** [comment]
LGTM

File: _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+title: 'SPEED CODERS OFFER ALIEN ROBOTS SCHOLARSHIPS: TGIF(79)'
+author: Cecily
+layout: blog-post-tgif
+comments: false
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it's time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. As usually, they're curated for you by [Cecily](https://twitter.com/skeskali)
```
Comment (varjmes) [inline _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md]: "As usually" should be "As usual"

File: _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
```
Comment (varjmes) [inline _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md]: should we turn these two links into markdown links? Right now they're just text.

File: _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+A couple of pieces from past Hoodie blogs that are still relevant today. I [Cecily](https://twitter.com/skeskali) am one of those non-coders who is happy to be a part of the team.
+
+[Non-Coders in Open Source](http://hood.ie/blog/non-coders-in-open-source-projects-or-on-aliens-confessions-of-a-non-coder-1.html)
+
+[Confessions of an Alien, Or: The Ultimate Guide to Attracting Non-Coding Contributors to Your Open Source Project](http://hood.ie/blog/confessions-of-an-alien-or-the-ultimate-guide-to-attracting-non-coding-contributors-to-your-open-source-project-2.html)
+
+[9 Open Source Robotics Projects](https://opensource.com/life/16/4/open-source-robotics-projects)
+
+Forbes has debuted [Well, Technically](http://www.forbes.com/sites/lillyknoepp/2016/04/12/well-technically-tech-and-the-people-who-love-it/#51ae31a060f0),  a new podcast that explores the experiences of people working in STEM. The show is hosted by Terri Burns and Sarah Kunst.
```
Comment (varjmes) [inline _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md]: too many spaces in ",  a"

**Comment by varjmes** [comment]
Just a few small things I noticed, happy when those are fixed. A great post, as always, Cecily <3

**Comment by gr2m** [comment]
@Charlotteis I’m out right now, could you please fix it yourself? You can also merge it and send out a tweet if you like :)

File: _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+author: Cecily
+layout: blog-post-tgif
+comments: false
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it's time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. As usually, they're curated for you by [Cecily](https://twitter.com/skeskali)
+
+Oh, and by the way: <b>our blog's RSS feed is back</b> – subscribe now on hood.ie or hood.ie/feed.xml to make sure you never miss a post! <3
```
Comment (janl) [inline _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md]: we don’t need this any more

**Comment by janl** [comment]
lgtm after comment, please run locally to check for formatting issues

File: _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md (hunk 2 above)
Comment (gr2m) [inline _posts/2016-04-15-speed-coders-offer-alien-robots-scholarships-tgif-79.md]: I’m not sure what you mean?

**Comment by varjmes** [comment]
sorry I missed your comment, gregor

**Pull request merged**